pkg/ride/converters.go

```diff
@@ -59,6 +59,8 @@ func transactionToObject(scheme byte, tx proto.Transaction) (rideObject, error)
 		return invokeScriptWithProofsToObject(scheme, transaction)
 	case *proto.UpdateAssetInfoWithProofs:
 		return updateAssetInfoWithProofsToObject(scheme, transaction)
+	case *proto.EthereumTransaction:
+		return ethereumInvokeScriptWithProofsToObject(scheme, transaction)
```

Comment:
`EthereumTransaction` can be `TransferTransaction` or `InvokeTransaction` Ride objects. You should handle both cases.
Note that `bodyBytes`and `proofs` Ride object fields should be filled by empty values in that transactions.